ng2-components/ng2-alfresco-viewer/src/components/viewer.component.ts

```diff
@@ -234,12 +233,21 @@ export class ViewerComponent {
     }
 
     /**
+     * check if the current file is a supported txt extension
+     *
+     * @returns {boolean}
+     */
+    private isText() {
+        return this.extension === 'txt' || this.mimeType === 'text/txt';
+    }
+
+    /**
      * check if the current file is  a supported extension
      *
      * @returns {boolean}
      */
     supportedExtension() {
```

Comment:
Better to have return type defined to avoid problems in the future. Guess it should be `boolean`